api/proto/teleport/legacy/types/types.proto

```diff
@@ -4237,6 +4237,8 @@ message ClusterAlertSpec {
     (gogoproto.stdtime) = true,
     (gogoproto.nullable) = false
   ];
+  // HostID is id of the node the alert applies to.
+  string HostID = 4 [(gogoproto.jsontag) = "host_id"];
```

Comment:
@EdwardDowling I would update the comment in the RFD to match, I don't think we want to keep this auth server specific.

Reply:
@rosstimothy The original idea (where the necessity for this field originated from) was to have a way to tell which auth server has expired license. But it can also be used to generate other node specific alerts in future, for example `node xxx is running outdated version` or something like this so we wanted to keep this field more or less generic - in this example you would just create multiple alerts, one for each node it applies to. We could make this a list in theory to potentially support creating alerts that apply to multiple nodes, not sure there's a use-case for that.

One thing I left a comment about previously though is host ID by itself does not really help much in helping users identify the node it applies to, I would change that to use a node name. @EdwardDowling

Reply:
Yeah we should rename it both here and in the RFD to `NodeName` - @EdwardDowling can you please do that? That way it would be consistent with how it's called in our config structs.

Reply:
I've swapped it over to instancename